openvpn-install.sh

```diff
@@ -146,48 +124,38 @@ prefetch: yes' >> /etc/unbound/unbound.conf
 	directory: "/etc/unbound"
 	trust-anchor-file: trusted-key.key
 	root-hints: root.hints
-	interface: 10.8.0.1
-	access-control: 10.8.0.1/24 allow
 	port: 53
 	num-threads: 2
-	use-caps-for-id: yes
 	harden-glue: yes
-	hide-identity: yes
-	hide-version: yes
-	qname-minimisation: yes
-	prefetch: yes' > /etc/unbound/unbound.conf
+	qname-minimisation: yes' >/etc/unbound/unbound.conf
```

Comment:
`harden-glue: yes` and `qname-minimisation: yes` are for all OS not only Arch. On `unbound.conf` manual they default to `yes` anyway.